ci/ci_test.bat

```diff
@@ -0,0 +1,44 @@
+@echo off
+
+setlocal
+
+REM Keep in mind to run ci/ci_setup.sh if you need to setup/clean up your environment before
+REM running the test suites here.
+
+REM Since we are using the system jruby, we need to make sure our jvm process
+REM uses at least 1g of memory, If we don't do this we can get OOM issues when
+REM installing gems. See https://github.com/elastic/logstash/issues/5179
+
+SET JRUBY_OPTS="-J-Xmx1g"
+SET SELECTEDTESTSUITE=%1
+SET /p JRUBYVERSION=<.ruby-version
```

Comment:
@jsvd we don't have this file in branches other than `master`, that may be a problem, right?
(I guess we could just add it there as well, that would be the cleanest?)